cleanlab_studio/studio/studio.py

```diff
@@ -383,3 +386,32 @@ def poll_cleanset_status(self, cleanset_id: str, timeout: Optional[int] = None)
 
         except (TimeoutError, CleansetError):
             return False
+
+    def autofix_dataset(
```

Comment:
allow string options to passed straight through into `_get_autofix_default_params()`

Reply:
Should be added now, clarified in the docs: 
https://github.com/cleanlab/cleanlab-studio/blob/afbe4a9bf8fba1c9eee8963bef042a57ef202090/cleanlab_studio/studio/studio.py#L377